scripts/zones/Mog_Garden/TextIDs.lua

```diff
@@ -2,16 +2,16 @@
 
 -- General Texts
 ITEM_CANNOT_BE_OBTAINED  = 6379; -- You cannot obtain the item <item> come back again after sorting your inventory
-ITEM_CANNOT_BE_OBTAINEDX = 6379;
+ITEM_CANNOT_BE_OBTAINEDX = 6378; -- (FIXME: Is this right??) You cannot obtain the <item>.
           ITEM_OBTAINED  = 6384; -- Obtained: <item>
         ITEM_OBTAINEDX   = 6390; -- You obtain <number> <item>!
            GIL_OBTAINED  = 6385; -- Obtained <number> gil
        KEYITEM_OBTAINED  = 6387; -- Obtained key item: <keyitem>
     NOT_HAVE_ENOUGH_GIL  = 6389; -- You do not have enough gil
-MOGLOCKER_MESSAGE_OFFSET = 7458;
+MOGLOCKER_MESSAGE_OFFSET = 7508; -- (FIXME: Is this right??) Your particular paid period of Mog Locker patronage has been extended until the following time, kupo!
```

Comment:
look at the green thumb moogles script to understand better the text IDs

On Sat, Aug 6, 2016 at 1:16 PM, thedax <notifications@github.com> wrote:

> In scripts/zones/Mog_Garden/TextIDs.lua
> <https://github.com/DarkstarProject/darkstar/pull/3307#discussion_r73790678>
> :
>
> >            ITEM_OBTAINED  = 6384; -- Obtained: <item>
> >          ITEM_OBTAINEDX   = 6390; -- You obtain <number> <item>!
> >             GIL_OBTAINED  = 6385; -- Obtained <number> gil
> >         KEYITEM_OBTAINED  = 6387; -- Obtained key item: <keyitem>
> >      NOT_HAVE_ENOUGH_GIL  = 6389; -- You do not have enough gil
> > -MOGLOCKER_MESSAGE_OFFSET = 7458;
> > +MOGLOCKER_MESSAGE_OFFSET = 7508; -- (FIXME: Is this right??) Your particular paid period of Mog Locker patronage has been extended until the following time, kupo!
>
> Had to take another guess here as well, since it had no comment or clues
> as to which mog locker line is required.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/3307/files/adc8d0cbe5e301a244f332b37d56a2ac0f4d5997..77e0bddf1c7f2d9095e72ec9d023cc695c1a0cf4#r73790678>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABGI_yD292P8rsuurjC551DfPQjuDl58ks5qdN1wgaJpZM4JdTzO>
> .
>